packages/components/src/Enumeration/Header/Header.component.js

```diff
@@ -20,7 +20,7 @@ function getAction(action, index) {
 	}
 
```

Comment:
you can add a test on disabled and if it is the case return null.
add if(action.disabled) { return null }

Reply:
test already done